services/package.json

```diff
@@ -16,7 +16,7 @@
     "postinstall": "cd .. && husky install"
   },
   "engines": {
-    "node": ">=14.x"
```

Comment:
Hoping to resolve issue in deployment here
https://github.com/Call-for-Code-for-Racial-Justice/Five-Fifths-Voter/runs/4374296936?check_suite_focus=true#step:11:66